integration_tests/transform/ecmascript-modules-support/yarn.lock

```diff
@@ -0,0 +1,549 @@
+# yarn lockfile v1
```

Comment:
I meant delete this whole file, not the first line in particular 🙂

Reply:
hehe